2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_setup.py:_flush():75] Loading settings from /home/atharva1511/.config/wandb/settings
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_setup.py:_flush():75] Loading settings from /home/atharva1511/Downloads/CS6910/assignment2/code2/wandb/settings
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Downloads/CS6910/assignment2/code2/train.py', 'program': 'train.py'}
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_init.py:_log_setup():405] Logging user logs to /home/atharva1511/Downloads/CS6910/assignment2/code2/wandb/run-20220322_024223-311guk5h/logs/debug.log
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_init.py:_log_setup():406] Logging internal logs to /home/atharva1511/Downloads/CS6910/assignment2/code2/wandb/run-20220322_024223-311guk5h/logs/debug-internal.log
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_init.py:init():439] calling init triggers
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'checkpoint_dir': 'models', 'train_file_path': '../20/dataset1.csv', 'val_file_path': '../20/dataset1.csv', 'cuda': False, 'wandb': False, 'do_train': True, 'do_val': True, 'do_predict': True, 'use_trained': True, 'batch_size': 1, 'epochs': 1000, 'val_batch_size': 1, 'output_num': 1, 'input_num': 48, 'num_ae_hidden1': 35, 'num_ae_hidden2': 25, 'num_ae_output': 10, 'num_hidden1': 13, 'num_hidden2': 18, 'learning_rate': 0.001, 'momentum': 0.9198}
2022-03-22 02:42:23,320 INFO    MainThread:15328 [wandb_init.py:init():492] starting backend
2022-03-22 02:42:23,320 INFO    MainThread:15328 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-22 02:42:23,324 INFO    MainThread:15328 [backend.py:ensure_launched():219] starting backend process...
2022-03-22 02:42:23,330 INFO    MainThread:15328 [backend.py:ensure_launched():224] started backend process with pid: 15350
2022-03-22 02:42:23,331 INFO    MainThread:15328 [wandb_init.py:init():501] backend started and connected
2022-03-22 02:42:23,334 INFO    MainThread:15328 [wandb_init.py:init():565] updated telemetry
2022-03-22 02:42:23,336 INFO    MainThread:15328 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-22 02:42:24,157 INFO    MainThread:15328 [wandb_run.py:_on_init():1759] communicating current version
2022-03-22 02:42:24,189 INFO    MainThread:15328 [wandb_run.py:_on_init():1763] got version response 
2022-03-22 02:42:24,189 INFO    MainThread:15328 [wandb_init.py:init():625] starting run threads in backend
2022-03-22 02:42:25,501 INFO    MainThread:15328 [wandb_run.py:_console_start():1733] atexit reg
2022-03-22 02:42:25,501 INFO    MainThread:15328 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-03-22 02:42:25,501 INFO    MainThread:15328 [wandb_run.py:_redirect():1611] Redirecting console.
2022-03-22 02:42:25,503 INFO    MainThread:15328 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-22 02:42:25,503 INFO    MainThread:15328 [wandb_init.py:init():664] run started, returning control to user process
2022-03-22 02:44:25,139 INFO    MainThread:15328 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 255
2022-03-22 02:44:25,140 INFO    MainThread:15328 [wandb_run.py:_restore():1674] restore
2022-03-22 02:44:27,662 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2022-03-22 02:44:28,021 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2022-03-22 02:44:28,203 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 3080
}

2022-03-22 02:44:28,306 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 20494
}

2022-03-22 02:44:28,409 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 20494
}

2022-03-22 02:44:28,520 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 20494
}

2022-03-22 02:44:28,623 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 20494
}

2022-03-22 02:44:28,725 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:28,827 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:28,930 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:29,033 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:29,136 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:29,240 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

2022-03-22 02:44:29,344 INFO    MainThread:15328 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20494
  total_bytes: 20494
}

